tutorials.md

```diff
@@ -21,7 +21,7 @@ Python.
 
 -   **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**
 
-    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
+    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github (too large to render preview)](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
```

Comment:
how does

   Github<sub>(too large to render preview)</sub>

look?

Reply:
Pretty good!